gcloud-java-pubsub/baseline/src/main/java/com/google/cloud/pubsub/spi/v1/SubscriberApi.java

```diff
@@ -280,6 +332,8 @@ public void close() throws IOException {
    * plus (`+`) or percent signs (`%`). It must be between 3 and 255 characters
    * in length, and it must not start with `"goog"`.
    * @param topic The name of the topic from which this subscription is receiving messages.
+   * The value of this field will be `_deleted-topic_` if the topic has been
```

Comment:
This comment is not clear to me. What has this to do with `create`? And why was this removed from `get` and `list` methods?